org-formation/075-security-hub/security-hub-suppress-infra.yaml

```diff
@@ -335,12 +336,14 @@ Resources:
       Timeout: 120
   BatchupdateEventInvokeConfig:
     Type: AWS::Lambda::EventInvokeConfig
+    DependsOn: SecurityHubFindingsSuppressionLambdaFct
```

Comment:
hmm, i wouldn't think DependsOn would be needed here or in BatchUpdateLambdaFctEventSourceEventSourceMapping since SecurityHubFindingsSuppressionLambdaFct is  already a reference in FunctionName param?

Reply:
Agree, I'm being over-explicit...